index.d.ts

```diff
@@ -7,12 +7,26 @@ type TestComponent = {
   mount(): Promise<void>;
 };
 
-export = kompot;
+type Provider<P> = {
+  component: React.ComponentType<P>;
+  props: P;
+  children: React.ComponentType<any>[];
+}
+
+export = kompot; 
 export as namespace kompot;
 
 declare namespace kompot {
-  function spy(id: string, getReturnValue?: (args: any[]) => any, stringifyArgs?: (args: any[]) => string[]): any
+  function spy(id: string, getReturnValue?: (args: any[]) => any, stringifyArgs?: (args: any[]) => string[]): () => any;
+  function spyOn(obj: object, methodName: string, spyId: string, stringifyArgs?: (args: any[]) => string[]): void;
   function kompotRequire(pathToComponent: string): { [key: string]: TestComponent };
   function init(): void;
-  function expect<T>(value: T): jest.Matchers<T> & Detox.Expect<T> & {notToHaveBeenCalled(): Promise<void>}
+  function expect<T>(value: T): jest.Matchers<T> & Detox.Expect<T> & {notToHaveBeenCalled(): Promise<void>};
+  function mockFetchUrl(url: string, cb: (url?: string) => Response): void;
```

Comment:
what is cb? anyways, the second parameter is the mock

Reply:
forgot to change it - cb stand for callback but it really isn't - will change